examples/tv-casting-app/darwin/TvCasting/TvCasting/CommissioningViewModel.swift

```diff
@@ -36,37 +69,45 @@ class CommissioningViewModel: ObservableObject {
     func prepareForCommissioning(selectedCommissioner: DiscoveredNodeData?) {
         if let castingServerBridge = CastingServerBridge.getSharedInstance()
         {
-            castingServerBridge.openBasicCommissioningWindow(DispatchQueue.main,
-                commissioningWindowRequestedHandler: { (result: Bool) -> () in
-                    DispatchQueue.main.async {
-                        self.commisisoningWindowOpened = result
+            castingServerBridge.setDacHolder(ExampleDAC(), clientQueue: DispatchQueue.main, setDacHolderStatus: { (error: MatterError) -> () in
+                DispatchQueue.main.async {
```

Comment:
this nest of calls is a bit difficult to parse, might be worthwhile splitting it out into a separate object

Reply:
Moved opening of the basic commissioning window to its own function and called it from here.